x-pack/solutions/search/test/functional_search/tests/search_getting_started.ts

```diff
@@ -113,6 +113,20 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
             const endpointValue = await testSubjects.getVisibleText('endpointValueField');
             expect(endpointValue).to.contain('https://');
           });
+
+          it('shows checkmark icon feedback when copy button is clicked', async () => {
```

Comment:
this is the only test you added that is being run, getting started is run as a part of the feature flags config for now, and you did not run the flakey test runner for those tests.